includes/managers/widgets.php

```diff
@@ -63,6 +63,7 @@ private function init_widgets() {
 			'common',
 			'inner-section',
 			'heading',
+			'testOne',
```

Comment:
Follow the coding of the existing widgets.
This should be `test-one`.
But: rather use the `table`.

@Mchris-hub

Reply:
Please update.